.all-contributorsrc

```diff
@@ -5,7 +5,8 @@
   "repoHost": "https://github.com",
   "files": [
     "README.md",
-    "packages/api/README.md"
+    "packages/api/README.md",
```

Comment:
Indenting? Run `yarn prettier --write .\.all-contributorsrc`

Reply:
Done in <!-- the charless --> 3153464